core/processor/tx.go

```diff
@@ -42,13 +43,19 @@ func DecodeTxNoValidation(payload []byte) (*Tx, error) {
 	if err := err.ErrorOrNil(); err != nil {
 		return nil, err
 	}
+	ttl := uint64(0)
```

Comment:
why is this only here (in null blockchain context) rather than (also) below in the `DecodeTx` function?

Reply:
You're right, should be added there, too. I suspect a `git checkout . && git checkout develop` without committing changes is to blame here